src/pe_reports/report_generator.py

```diff
@@ -55,6 +56,16 @@ def load_template():
     return prs
 
 
+def load_customers():
+    """Export PowerPoint report set to output directory."""
+    try:
+        file = open(CUSTOMERS)
+        names_obj = json.load(file)
```

Comment:
You are manually opening a file and not closing it when finished. This will handle it automatically for these purposes.

```suggestion
        with open(CUSTOMERS) as customers_file:
            names_obj = json.load(customers_file)
```